ambari-common/src/main/python/resource_management/libraries/execution_command/module_configs.py

```diff
@@ -23,18 +23,40 @@
 
 class ModuleConfigs(object):
   """
-  This class maps to "/configurations" block in command.json which includes configuration information of a service
+  This class maps to "/configurations" and "/configurationAttributes in command.json which includes configuration information of a service
   """
 
-  def __init__(self, config):
+  def __init__(self, config, configAttributes):
```

Comment:
Nit-picking: Rename "config" to "configs" (plural instead of singular)

def __init__(self, **configs**, configAttributes):

Reply:
Done